lib/carrierwave/mongoid/version.rb

```diff
@@ -1,5 +1,5 @@
 module Carrierwave
   module Mongoid
-    VERSION = "1.3.0"
+    VERSION = "2.0.0"
```

Comment:
Not sure how you guys would want to handle a version update, as part of this PR or separately. Also not sure about the target version number. As I understand semver, a major version bump would be in order due to the backwards incompatible change.